docs/pages/docs/guides/images-and-files.mdx

```diff
@@ -0,0 +1,272 @@
+      generateUrl: path => `${baseUrl}/images${path}`,
+      // The route that will be created in Keystone's backend to serve the images
+      serverRoute: {
+        path: '/images',
+      },
+      storagePath: 'public/images',
+    },
+    // The key here will be what is referenced in the file field
+    my_s3_files: {
+      // Files that use this store will be stored in an s3 bucket
+      kind: 's3',
+      // This store is used for the file field type
+      type: 'file',
+      // The S3 bucket name pulled from the S3_BUCKET_NAME environment variable above
+      bucketName,
+      // The S3 bucket region pulled from the S3_REGION environment variable above
+      region,
+      // The S3 Access Key ID pulled from the S3_ACCESS_KEY_ID environment variable above
+      accessKeyId,
+      // The S3 Secret pulled from the S3_SECRET_ACCESS_KEY environment variable above
+      secretAccessKey
+      // The S3 links will be signed so they remain private
+      signed: { expiry: 5000 },
+    },
+  },
+});
+```
+
+!> You can define as many stores as required for your use case
+
+## Field types - `image` and `file`
+
+Once you have your `storage` configuration in Keystone you can then use the `image` and `file` [field types](../apis/fields).
+
+Within an existing [list](../apis/schema) use the `image` of `file` field as follows:
+
+```typescript
+lists: {
+  user: list({
+    fields: {
+      /** other fields */
+      avatar: image({ storage: 'my_local_images' }),
+      someFile: file({ storage: 'my_s3_files' }),
+    }
+  })
+}
+```
+
+!> The `storage` name here relates to the `storage` defined in your `keystone.ts`
+
+## Relationships - ie Image Gallery
+
+If you want your images (and files) to be reused and accessible across multiple difference lists or in a [Document Field](./document-fields). You can set up a Gallery list and use relationships to reference this.
+You might asl want to preserve an asset after 'deletion', with relationships you can instead remove the relationship and cleanup the assets separately.
+
+For example you might have a schema that looks like this:
+
+```typescript
+// schema.ts
+import { list } from '@keystone-6/core';
+import { text, image } from '@keystone-6/core/fields';
+
+export const lists = {
+  Image: list({
+    fields: {
+	  name: text({
+		  validation: {
+	    isRequired: true,
+		  },
+	  }),
+	  altText: text(),
+	  image: image({ storage: 'my_local_images' }),
+    }
+  }),
+  Page: list({
+    fields: {
+      name: text(),
+      context: text(),
+      images: relationship({ ref: 'Image', many: true })
+    }
+  })
+};
+```
+
+## A note on `Content-Type`
+
+When serving files, either from local or S3, Keystone uses the `Content-Type` of `application/octetstream`. For images, however, the `Content-Type` is set from the mime-type configured by the file extension.
+This means that for images the extension can be trusted, however, for files it can not.
```

Comment:
```suggestion
- When serving **files**, Keystone uses `application/octetstream` for the `Content-Type`.
- When serving **images** the `Content-Type` is set from the mime-type configured by the file extension.

This means that for images the extension can be trusted, but for files it can not. This applies to both `local` and `S3` options.
```